### ISSUE TITLE
read_rgba_from_png wrong output

### ISSUE THREAD

**Issue by DanielZFLiu** [dilevin/computer-graphics-raster-images#52]
adding std::cout << rgba.size(); after line 35 in main.cpp yields me 0. Is there something wrong with the function?

**Comment by DanielZFLiu** [comment]
turns out that the relative paths on line23 - line26 is wrong; maybe (probably) I built the project wrong.

**Comment by sannat17** [comment]
Could it be that you are running the code from the root of the project directory (i.e. by calling `build/raster`)? Could you try calling `./raster` from inside the build directory and see if this issue still persists. Also, make sure that your build directory is located within the root of the project directory and not somewhere else.

PS: In my opinion, hardcoding paths in a C++ function is probably not the best way to do things anyway. It would have been much cleaner if paths (for input and output directories) were instead given as arguments and instructions showed how to use the compiled binary.

**Comment by wenzhi-guo** [comment]
Yes, it indeed looks like a problem with the path of the input images. In general, the "data" folder should be one level above the raster.exe file for building on windows (see tutorial slides for a potential solution for this).

**Comment by DanielZFLiu** [comment]
Thank you! I didn't even notice the slides were posted.
